templates/default/st2.conf.erb

```diff
@@ -3,30 +3,46 @@
 
 [api]
 # Host and port to bind the API server.
-host = 0.0.0.0
-port = 9101
-logging = <%= @conf_root %>/st2api/console.conf
+host = <%= @api_host %>
+port = <%= @api_port %>
+logging = /etc/st2/logging.api.conf
 mask_secrets = <%= @mask_secrets %>
 # allow_origin is required for handling CORS in st2 web UI.
 # allow_origin = http://myhost1.example.com:3000,http://myhost2.example.com:3000
 allow_origin = <%= @allow_origin %>
 
+[stream]
+logging = /etc/st2/logging.stream.conf
+
 [sensorcontainer]
-logging = <%= @conf_root %>/st2reactor/console.conf
+logging = /etc/st2/logging.sensorcontainer.conf
```

Comment:
Yeah, included explanation in Commit message:

> You can't overwrite `conf_root` from `/etc/st2` to anything else, since it's globally hardcoded setting.

So it's hardcoded in packages, you can't change it and it's used in so many places.
Allowing users to edit `conf_root` brings confusion and broken installation.